hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OmUtils.java

```diff
@@ -268,6 +268,7 @@ public static boolean isReadOnly(
     case SnapshotDiff:
     case ListSnapshotDiffJobs:
     case TransferLeadership:
+    case TestUnknownCommand:
```

Comment:
Hi, I have set it as a read-only request to avoid the need for explicit request/response handling associated with a write request. 

Not setting it under either option is causing the `TestOzoneManagerRatisServer#testIsReadOnlyCapturesAllCmdTypeEnums` test case to fail where we are expected to either categorize the Type Enums as a read request or a write request.

Do you suggest we modify the test case to avoid this one request (since we are reserving this `cmdType` for testing purposes only)?

Reply:
>> Do you suggest we modify the test case to avoid this one request (since we are reserving this cmdType for testing purposes only)? 

Yes, you could also add a comment to point out this specific case. 
The test on both ratis and non-ratis env should be useful.

This PR could prevent hacked client request attacks to some extent. (possibly)

Reply:
Understood, thanks @Xushaohong.